syntax = "proto3";

message BlockRaw {
  string signature = 1;
  string item_id = 2;
  string next_leader = 3;
  uint32 transaction_count = 4;
  string type = 5;
  string version = 6;
  string peer_id = 7;
  uint32 number = 8;
  string merkle_root_hash = 9;
  string item_timestamp = 10;
  string hash = 11;
  string parent_hash = 12;
  uint64 timestamp = 13;
}
